hw-model/src/lib.rs

```diff
@@ -633,10 +633,8 @@ pub trait HwModel: SocManager {
         self.soc_ifc()
             .fuse_owner_pk_hash()
             .write(&fuses.owner_pk_hash);
-        self.soc_ifc()
-            .fuse_fmc_key_manifest_svn()
-            .write(|_| fuses.fmc_key_manifest_svn);
-        self.soc_ifc().fuse_runtime_svn().write(&fuses.runtime_svn);
+        self.soc_ifc().fuse_fmc_key_manifest_svn().write(|_| 0); // FMC SVN deprecated
```

Comment:
For a future PR to caliptra-rtl, in the RDL we should maybe update this to be unpopulated so we don't generate any code for it.

Reply:
Ack.